node_shell.py

```diff
@@ -1035,6 +1092,9 @@ def load_ipy(cmd2_app, py_bridge):
             """
             from aiida.cmdline.utils.shell import get_start_namespace
             from IPython import embed
+            from traitlets.config import get_config
+            conf = get_config()
+            conf.InteractiveShellEmbed.colors = "Linux"
```

Comment:
I don't know if this is the reason, but now I get white text, and my default shell has white background... Is there a way to just use the same default as ipython would do?